### DISCUSSION TITLE
Migration path from Beta to Release

### DISCUSSION THREAD

**Discussion by robin-thoni** [music-assistant/support#2327] [Q&A] [answered]
Is there a recommended migration path from the Beta Home Assistant Add-on to the Release one? Do we have to reconfigure providers, players and Home Assistant players entities?

Thanks!

**Comment by OzGav** [comment, marked as answer]
Yes at this stage you need to redo your setup.

**Reply by robin-thoni** [reply]
Thanks for the confirmation

**Comment by lor74** [comment]
Hi @OzGav is still the only solution (redo setup)? Now that 2.3 for me is stable I would like to move to stable branch
Thanks in advance.

**Reply by OzGav** [reply]
Yes that is still the situation.
